apps/emqx_connector_aggregator/src/emqx_connector_aggreg_sup.erl

```diff
@@ -5,28 +5,28 @@
 -module(emqx_connector_aggreg_sup).
```

Comment:
As a super-generic supervisor, in my opinion this module currently serves no purpose as part of `emqx_connector_aggregator` app. Maybe just moving it back and renaming to `emqx_bridge_s3_sup` would be easier?